client/src/Components/Widget0.js

```diff
@@ -52,29 +52,22 @@ class Widget0 extends React.Component {
     }
 
     componentDidMount(){
-        
-    //    let request = new Request(`http://trackleaders.com/spot/transam19/fullfeed.xml`);
-    //    let uri = "http://trackleaders.com/spot/transam19/fullfeed.xml";
-            
-    //    let h = new Headers({
-    //     'Access-Control-Allow-Origin':'*',
-    //     'Content-Type': 'multipart/form-data'
-    // });
-        
-    //     let req =  {
-    //         method: 'POST',
-    //         headers: h,
-    //         mode: 'cors'
-    //     };
-
-    //    fetch(uri,req)
-    //    .then(response=> response.json())
-    //    .catch(function(error) {
-    //      console.log('Looks like there was a problem: \n', error);
-    //    });
       
-    }
-
+        this.trackData();
+     }
+ 
+     trackData(){
+ 
+         setInterval(function(){
+             fetch('/api/trackleads')
+             .then(res => res.json())
```

Comment:
the format of the response data from the server isn't in `geojson` format.  You'll want to make a function to organize this data in geojson format as a feature collection to add to Mapbox as a source, i.e.

```
{
  "type": "FeatureCollection",
  "features": [
    {
      "type": "Feature",
      "properties": {
        "speed": 6
      },
      "geometry": {
        "type": "Point",
        "coordinates": [
          -92.8125,
          36.59788913307022
        ]
      }
    }
  ]
}
```

The raw data looks like this from the server:
```
{
    "trackleaders_aggregate_feed": {
        "trackleaders_feed": [
            {
                "trackleaders_racer_ID": [
                    "1"
                ],
                "message": [
                    {
                        "id": [
                            "1211519923"
                        ],
                        "esn": [
                            "0-3158346"
                        ],
                        "esnName": [
                            "0-3158346"
                        ],
                        "messageType": [
                            "UNLIMITED-TRACK"
                        ],
                        "messageDetail": [
                            ""
                        ],
                        "timestamp": [
                            "2019-06-06T02:16:21.000Z"
                        ],
                        "timeInGMTSecond": [
                            "1559787381"
                        ],
                        "latitude": [
                            "45.67239"
                        ],
                        "longitude": [
                            "-116.30511"
                        ],
                        "batteryState": [
                            "GOOD"
                        ],
                        "elevation": [
                            "505.000000"
                        ]
                    },
                    {
                        "id": [
                            "1211518176"
                        ],
                        "esn": [
                            "0-3158346"
                        ],
                        "esnName": [
                            "0-3158346"
                        ],
                        "messageType": [
                            "UNLIMITED-TRACK"
                        ],
                        "messageDetail": [
                            ""
                        ],
                        "timestamp": [
                            "2019-06-06T02:11:18.000Z"
                        ],
                        "timeInGMTSecond": [
                            "1559787078"
                        ],
                        "latitude": [
                            "45.66441"
                        ],
                        "longitude": [
                            "-116.29445"
                        ],
                        "batteryState": [
                            "GOOD"
                        ],
                        "elevation": [
                            "466.000000"
                        ]
                    },
```